README.md

```diff
@@ -221,23 +221,25 @@ You need to add the following dependencies to your HTML:
             import { Diagram  } from './dist/quick-erd.js';
 
             new Diagram( toERD(
-`departments /insert 2
+`
+departments
     name /nn
     location
     country
-    employees /insert 4
-        name /nn vc50
-        email /lower
-        cost center num
-        date hired
-        job vc255
+employees
+    departments_id /fk departments
```

Comment:
This is the same as if employees were indented, why change it?

Reply:
If employees is indented and departments_id column is not defined, the line of the diagram is not correct, because toERD returns: 
source: "departments"
source_id:  "id"
target: "employees"
target_id: "department_id"
but the target_id  (department_id) is missing in employees